app.resp.RESP.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id": "app.resp.RESP",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "resp",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=~/.rdm:create",
```

Comment:
Just to double-check, this is because RESP still uses `~/.rdm` for configuration files?

Reply:
Yes, for backward compatibility.